lib/workers/repository/onboarding/pr/index.js

```diff
@@ -48,7 +48,7 @@ async function ensureOnboardingPr(config) {
       prBody = config.global.prBanner + '\n\n' + prBody;
     }
     if (config.global.prFooter) {
-      prBody = prBody + '\n---\n\n' + config.global.prFooter + '\n';
```

Comment:
The `---` separator is to ensure that the footer content shows up under it's own visually distinctive section. 

If the pull request has errors or warnings, then this change will cause the `#### About merge conflicts` title to appear within the [section for errors or warnings](https://github.com/renovateapp/renovate/blob/23d0290384b53fcb66a63cfd148f3fd1dd6c0a01/lib/config/templates/default/pr-body.hbs).

It would make sense to either keep the `---` separator, or to place one in the template file?